app/internal/page/set_token.go

```diff
@@ -0,0 +1,38 @@
+package page
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/signin"
+)
+
+func SetToken(c signin.Client, appPublicURL, clientID, JTI string) http.HandlerFunc {
```

Comment:
`Callback` or `SigninCallback` or `AuthCallback` might be better names for this

Reply:
`SigninCallback` ✅